### DISCUSSION TITLE
JSON API to drive website

### DISCUSSION THREAD

**Discussion by judeibe** [community-scripts/ProxmoxVE#9] [Ideas]
If you ever visited brew.sh you would notice it was a static site and uses JavaScript to look through GitHub. It's powered by a json file in the repo that has all the details for the site.

I think this can be used to make the site more maintainable and also. This would require the json file be a required component of any new script.

**Comment by MickLesk** [comment]
Is this a call for the Website?
Here is the repo for the Page:
https://github.com/community-scripts/Proxmox

**Reply by judeibe** [reply]
Both as the json should be in this repo. So it can be tied to the script PR.

**Comment by bramsuurdje** [comment]
working on it. fully agree with your points
